modules/debts_api/spec/lib/debt_api/v0/fsr_form_transform/personal_data_calculator_spec.rb

```diff
@@ -11,20 +11,23 @@
     let(:post_transform_fsr_form_data) do
       get_fixture_absolute('modules/debts_api/spec/fixtures/pre_submission_fsr/post_transform')
     end
-
-    def get_personal_data
+    let(:transformer_data) do
       transformer = described_class.new(pre_transform_fsr_form_data)
-      @data = transformer.get_personal_data
+      transformer.get_personal_data
     end
 
     describe '#get_personal_data' do
-      before do
-        get_personal_data
-      end
-
       it 'gets personal data correct' do
         expected_personal_data = post_transform_fsr_form_data['personalData']
-        expect(expected_personal_data).to eq(@data)
+        expect(expected_personal_data).to eq(transformer_data)
+      end
+
+      it 'returns empty string for addressLine2 and addressLine3' do
```

Comment:
Could you add an `expect` for `addressLine3` to match the description "and addressLine3"

`expect(calculator_data['address']['addresslineThree']).to eq('')`

Reply:
For sure, nice catch.